src/main/java/service/impl/FileReaderImpl.java

```diff
@@ -0,0 +1,25 @@
+package service.impl;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import service.FileReader;
+
+public class FileReaderImpl implements FileReader {
+    @Override
+    public List<String> read(String fileName) {
+        List<String> listOfLines = new ArrayList<>();
+        File file = new File(fileName);
+        try (BufferedReader reader = new BufferedReader(new java.io.FileReader(file))) {
+            String value;
+            while ((value = reader.readLine()) != null) {
+                listOfLines.add(value);
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Cannot read the file " + file.getPath());
+        }
+        return listOfLines;
+    }
+}
```

Comment:
The class `FileReaderImpl` should be placed in a package that reflects its purpose. Consider moving this class into a `service.impl` package to maintain a clean and organized project structure.